### ISSUE TITLE
"Whoops. Looks like an internal server error. Search your server logs for request ID...."

### ISSUE THREAD

**Issue by MitchEff** [prismagraphql/prisma#2855]
tl;dr: Do these logs... exist? 

I can see a couple other people have asked about this, and are told _"Simple! Just check your Docker logs"_.

Now, the tutorial on [HowToGraphQL](https://www.howtographql.com/react-apollo/0-introduction/) doesn't mention Docker at all. I've installed Docker specifically to try and find these logs, to which I've read _"Simple! Just pop in your container ID"_. 

What's that? Doesn't seem to relate to 'service', 'cluster' or 'endpoint'.

There might be something super simple I'm missing, but if I am then I'm sure many are too. Where does Docker come into it (if everything appears to be running fine without it), where are the logs being hidden and where does one find a container ID?

**Comment by marktani** [comment]
Ok, I see!

You can _either_ deploy a service

1. to a demo server (there is one in EU and one in US)
2. to a local Docker container running Prisma
3. to a remote machine running a Docker container running Prisma

The instructions you found refer to options 2. and 3. You would be able to run `docker ps` to find out the container id of the Prisma container, and subsequently run `docker logs -f <container id>` to access its logs.

For the demo server, you cannot easily access the server logs yourself. Please reach out to `support@prisma.io` including your request id and an error description so we can provide you with specific information to the request id.

Thanks! 🙂

**Comment by Param-Harrison** [comment]
@marktani this is the issue I get in my docker logs,
```
org.postgresql.util.PSQLException: ERROR: column User.username does not exist
prisma_1    |   Hint: Perhaps you meant to reference the column"User.userName".
```
How to rectify it? I am debugging it for last two days with no help, everywhere I see `docker logs -f <container id>` as the answer which is useful to debug but useless to solve the issue

**Comment by marktani** [comment]
@Param-Harrison, please share your setup and error in a new bug report using this template https://github.com/prismagraphql/prisma/issues/new?template=bug_report.md :slightly_smiling_face:

**Comment by Param-Harrison** [comment]
thanks for the reply man @marktani. I will do.

**Comment by nugoo1** [comment]
Is there a solution to this?? Im getting this same error!

**Comment by Khareta** [comment]
We're facing this issue right now, the logs could be better. Any suggestions?
